conda/install.py

```diff
@@ -261,16 +261,31 @@ def rm_rf(path, max_retries=5, trash=True):
         # islink('/path/to/dead-link') is True.
         try:
             os.unlink(path)
+            return
         except (OSError, IOError):
```

Comment:
My guess is that we still might need to tune some of this exception catching, either tightening it down or loosening it up either way.  But I'm fine with iterating until this is exactly right.